components/radiobuttons/index.ts

```diff
@@ -0,0 +1,62 @@
+import m from "mithril";
+
+
+export type OptionValue = {
+    title?: string, 
+    value: string,
+    icon?: string,
+    disabled?:boolean,
+    id?: string
+}
+
+export type Attrs = {    
+    title: string,
+    value?: string, 
+    options: Array<OptionValue>
+    disabled?: boolean, 
+    onchange?: (event) => void;
+    onblur?: (event) => void;
+    onfocus?: (event) => void;
+}
+
+export type State = {
+    $container: HTMLElement;
+}
+
+export default {
+    view(vnode) {
+        const { value, options, disabled } = vnode.attrs;
+        return m(".mmf-radio-btn-container", {
+                disabled,
+                oncreate: _vnode => (this.$container = _vnode.dom as HTMLElement),
+            },
+            options.map((option: OptionValue) => {
+                const icon = option.icon ? 
+                    m("span.mmf-icon", {  
+                        value: option.value,
+                    }, option.icon) 
+                    : undefined;
+
+                return m(".mmf-radio-btn",
```

Comment:
1. using divs here will prevent tabbing through form-elements for this kind of input-type. We should use html-buttons here, which support states like focus and tab-index
2. using html-buttons its getting obvious that we can reuse the button component to share styles, layout and behaviour
3. introducing buttons as form-element here, will add support for keyboard input